src/components/CreateHappyThought.css

```diff
@@ -0,0 +1,85 @@
+.create-thought__container {
+  display: flex;
+  flex-flow: row wrap;
+
+  border: 2px solid var(--black);
+  box-shadow: -0.5rem 0.5rem 0 0 var(--black);
+  background-color: var(--gray-200);
+
+  padding: 1.25rem;
+}
+
+.create-thought__title {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 0;
+}
+
+.create-thought__form {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  width: 100%;
+
+  label,
+  output {
+    font-size: 0.875rem;
+    font-weight: 600;
+  }
+
+  label {
+    margin-bottom: 1rem;
+  }
+
+  textarea {
+    font-family: "IBM Plex Mono", "Courier New", "Courier", sans-serif;
+    font-size: 1rem;
+    padding: 0.875rem;
+    resize: none;
```

Comment:
⭐

Reply:
I'm always really impressed with how well-written and organized is your code. 

Some suggestions for improvement could be:
1. Instead of just logging errors,  show a user-friendly error message when posting fails.
2. Add an aria-label to the textarea for screen reader support.

Overall, great work 😊